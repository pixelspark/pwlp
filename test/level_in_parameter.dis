0000.	11	PUSHB	[03]
0002.	11	PUSHB	[05]
0004.	21	PEEKB	1
0005.	11	PUSHB	[01]
0007.	80	BINARY	ADD
0008.	11	PUSHB	[ff]
0010.	85	BINARY	AND
0011.	22	PEEKB	2
0012.	11	PUSHB	[ff]
0014.	85	BINARY	AND
0015.	74	UNARY	SHL8
0016.	86	BINARY	OR
0017.	22	PEEKB	2
0018.	11	PUSHB	[ff]
0020.	85	BINARY	AND
0021.	74	UNARY	SHL8
0022.	74	UNARY	SHL8
0023.	86	BINARY	OR
0024.	e3	USER	set_pixel
0025.	01	POP	1
0026.	e4	USER	blit
0027.	01	POP	1

