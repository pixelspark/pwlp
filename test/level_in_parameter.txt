a=3;
set_pixel(5, a+1, a, a);
blit;