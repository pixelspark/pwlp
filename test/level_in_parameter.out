000000 000000 000000 000000 000000 040303 000000 000000 000000 000000 
